### DISCUSSION TITLE
Add multiple carets on double ⌥ Option with arrow keys (similar to Intellij)

### DISCUSSION THREAD

**Discussion by doichev-kostia** [zed-industries/zed#22282] [Feature Requests]
There is currently functionality to put the multi-cursor using mouse (option + click), but is there an option to do that using the keyboard without binding to a specific word (`cmd-d` on macOS)?

You can see the behaviour in Intellij by pressing the `option` key twice https://www.jetbrains.com/help/idea/multicursor.html#add-carets-above-or-below-the-current-caret

https://www.jetbrains.com/help/idea/settings-smart-keys.html "Add multiple carets on double ⌥ Option with arrow keys"

**Comment by doichev-kostia** [comment]
WIth Vim mode similar functionality can be achieved using `ctrl + v`

**Comment by 5brian** [comment]
```
      "shift-alt-up": "editor::AddSelectionAbove", // Insert Cursor Above
      "shift-alt-down": "editor::AddSelectionBelow", // Insert Cursor Below
```

this is from vscode base keymap, you can search `editor::AddSelectionAbove` in your default settings

**Reply by doichev-kostia** [reply]
Thank you!

```json
{
  "context": "Editor",
  "bindings": {
    "alt-alt-up": "editor::AddSelectionAbove",
    "alt-alt-down": "editor::AddSelectionBelow",
  }
}
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
